### ISSUE TITLE
Deleting .import files wipes every tres file in the project of all resources and settings

### ISSUE THREAD

**Issue by chucklepie** [godotengine/godot#45280]
**Godot version:**
3.2.3

**OS/device including version:**
Linux

**Issue description:**
I closed Godot, deleted my .import folder so I could clean it up. On starting Godot I noticed everything was broken. On checking my local repository I noticed that every single tres file has been reset:

![image](https://user-images.githubusercontent.com/12863685/104893377-42155880-596b-11eb-8032-1a1b89d2f0b0.png)

**Steps to reproduce:**
delete import folder, open project.

**Comment by chucklepie** [comment]
I've actually seen this before, but it only happened to one file and I recreated it. But here, it's every single one.

**Comment by akien-mga** [comment]
> Steps to reproduce:
delete import folder, open project.

Could you provide a minimal project that triggers the issue? I've deleted `.import/` folders hundreds of times and never experienced this.

**Comment by chucklepie** [comment]
Yes. I thought I'd just raise it in case it was a known problem I didn't
find.

It's happened once before to me. I suspect there's more to it, probably a
sequence of events, but I'll try and narrow it down.

What I have noticed, but probably not related is godot randomly generates
missing resource and rid errors when importing lots of files, even though
they imported ok.

**Comment by chucklepie** [comment]
So, to help, given I touched the tres files and all the affected ones are
in my tilemap, can somebody let me know what set of actions or methods in
godot will delete png files from tres file but leave the number of frames,
as per my image?

**Comment by fabiodellaria** [comment]
Did someone fixed this issue? Or at least found a workaround?

**Comment by Calinou** [comment]
> Did someone fixed this issue? Or at least found a workaround?

If you can reproduce this issue on Godot 3.5 or later, please upload a minimal reproduction project, as we still don't have any.

**Comment by fabiodellaria** [comment]
I tried to reproduce the issue in a little project without success, but I found a workaround (at least for me), just close Godot after the new import of the assets and after reopen it... after the last opening Godot for me works all.
